tools/compare.go

```diff
@@ -15,125 +15,291 @@
 package tools
 
 import (
-	"encoding/json"
+	"context"
 	"fmt"
-	"io"
-	"strconv"
-	"sync"
-
-	"github.com/google/go-cmp/cmp"
 	"github.com/spf13/cobra"
 	"github.com/streamingfast/bstream"
+	"github.com/streamingfast/cli"
 	"github.com/streamingfast/dstore"
+	"github.com/streamingfast/eth-go/rpc"
 	pbeth "github.com/streamingfast/firehose-ethereum/types/pb/sf/ethereum/type/v2"
+	"github.com/streamingfast/substreams/block"
+	"go.uber.org/multierr"
+	"google.golang.org/protobuf/proto"
+	"io"
+	"math"
+	"os"
+	"os/exec"
+	"strconv"
+	"sync"
 )
 
 var compareBlocksCmd = &cobra.Command{
-	Use:   "compareblocks <blocks_store_a> <blocks_store_b> <prefix>",
-	Short: "Checks for any differences between merge files of two different stores. (To compare the output of two different instrumentations, for example)",
-	Args:  cobra.ExactArgs(3),
-	RunE:  compareBlocksE,
+	Use:   "compare-blocks <expected_bundle> <actual_bundle> [<block_range>]",
+	Short: "Checks for any differences between two block stores between a specified range. (To compare the likeness of two block ranges, for example)",
+	Long: cli.Dedent(`
+		compare-blocks takes in two paths to stores of merged blocks and a range specifying the blocks you want to compare (written as: start-finish).
```

Comment:
Start with a capital, so in this case I would start with `The 'compare-blocks'`. Also, wrap "code/command" like element in `'`, so `'compare-blocks'`, `'<start>:<finish>'`, etc.

Reply:
Show the range as `:` (no need to say we accept `-`)